### ISSUE TITLE
PermissionDeniedError with Visualization Creator

### ISSUE THREAD

**Issue by kim-sardine** [kubeflow/pipelines#2883]
**What happened:**

When I tried to `Generate Visualization` with TFDV on running pipeline, I got static HTML with error as below.

I can run all pipeline just normally. I can get object in bucket on pipeline running time. but this visualization step only have a problem

```
---------------------------------------------------------------------------
PermissionDeniedError                     Traceback (most recent call last)
<ipython-input-18-e338bd9087f5> in <module>
     21 # source
     22 
---> 23 train_stats = tfdv.generate_statistics_from_csv(data_location=source)
     24 
     25 tfdv.visualize_statistics(train_stats)

/usr/local/lib/python3.5/dist-packages/tensorflow_data_validation/utils/stats_gen_lib.py in generate_statistics_from_csv(data_location, column_names, delimiter, output_path, stats_options, pipeline_options)
    152     skip_header_lines = 1 if column_names is None else 0
    153     if column_names is None:
--> 154       column_names = _get_csv_header(data_location, delimiter)
    155     _ = (
    156         p

/usr/local/lib/python3.5/dist-packages/tensorflow_data_validation/utils/stats_gen_lib.py in _get_csv_header(data_location, delimiter)
    290       different headers.
    291   """
--> 292   matched_files = tf.gfile.Glob(data_location)
    293   if not matched_files:
    294     raise ValueError(

/usr/local/lib/python3.5/dist-packages/tensorflow/python/lib/io/file_io.py in get_matching_files(filename)
    359     errors.OpError: If there are filesystem / directory listing errors.
    360   """
--> 361   return get_matching_files_v2(filename)
    362 
    363 

/usr/local/lib/python3.5/dist-packages/tensorflow/python/lib/io/file_io.py in get_matching_files_v2(pattern)
    387           # Convert the filenames to string from bytes.
    388           compat.as_str_any(matching_filename)
--> 389           for single_filename in pattern
    390           for matching_filename in pywrap_tensorflow.GetMatchingFiles(
    391               compat.as_bytes(single_filename), status)

/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/errors_impl.py in __exit__(self, type_arg, value_arg, traceback_arg)
    526             None, None,
    527             compat.as_text(c_api.TF_Message(self.status.status)),
--> 528             c_api.TF_GetCode(self.status.status))
    529     # Delete the underlying status object from memory otherwise it stays alive
    530     # as there is a reference to status from this from the traceback due to

PermissionDeniedError: Error executing an HTTP request: HTTP response code 403 with body '{
  "error": {
    "code": 403,
    "message": "Primary: /namespaces/[PROJECT_ID].svc.id.goog with additional claims does not have storage.objects.list access to [BUCKET_NAME].",
    "errors": [
      {
        "message": "Primary: /namespaces/[PROJECT_ID].svc.id.goog with additional claims does not have storage.objects.list access to [BUCKET_NAME].",
        "domain": "global",
        "reason": "forbidden"
      }
    ]
  }
}
'
	 when reading gs://[BUCKET_NAME]/[PATH_TO_FILE]
```

It seems like I didn't set some permission to `identity-namespace`, but I don't know where I can set this.

**What steps did you take:**

I'm using latest kubeflow on GCP (0.7.0) and I deployed it using official guide [Deploy using CLI](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/)

and I updated `pipeline-visualization-service` to `version : 0.1.31` because of [this issue](https://github.com/kubeflow/pipelines/issues/2806)

**Comment by aaaaahaaaaa** [comment]
Same problem here.

**Comment by Bobgy** [comment]
In Kubeflow deployment, visualization server isn't bound with an Google Service Account, so it doesn't have any permission.

Recommend doing so by https://github.com/kubeflow/pipelines/blob/5391e88fbca11d2eb70cfc42d06202770d0a174c/manifests/kustomize/wi-utils.sh#L27

**Comment by Bobgy** [comment]
If `kf-user` works for you, then we can contribute that change back to kubeflow/manifest

**Comment by Bobgy** [comment]
Already fixed in KF 1.0.2 by using `kf-user`
